bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/internal/profiles/SystemProfileFactory.java

```diff
@@ -179,7 +181,7 @@ public SystemProfileFactory(final @Reference ChannelTypeRegistry channelTypeRegi
                 case STATE:
                     return DEFAULT;
                 case TRIGGER:
-                    return null;
+                    return TIMESTAMP_TRIGGER;
```

Comment:
Pfew, I am not so sure if this profile should really become the generally suggested one...
I was just [looking into profiles](https://github.com/openhab/openhab-webui/issues/886#issuecomment-844381424) today and was thinking that a "map-to-string" profile (need to find a better name) would probably make sense as a default profile for trigger channels. Wdyt?

Reply:
I reverted this change.

Reply:
If I understand your idea correctly you like to add a `TriggerProfile` which converts the triggered event into a `StringType`? 

What about simply "toString"?